src/Resolution.ts

```diff
@@ -552,6 +552,23 @@ export default class Resolution {
     );
   }
 
+  /**
+   * @param domain - domain name
+   * @param network - network name
+   * @param token - token ticker
+   * @returns An owner address of the domain
+   */
+  async getAddress(
+    domain: string,
+    network: string,
+    token: string,
+  ): Promise<string | null> {
+    domain = prepareAndValidateDomain(domain);
+    return this.callServiceForDomain(domain, (service) =>
+      service.getAddress(domain, network, token),
+    );
+  }
```

Comment:
I'd defer that until we release a major version with a new json record and underline implementation of those APIs. I intended to release this as a beta API since we are still not sure if the mapping works correctly